### ISSUE TITLE
Transform data to allen space

### ISSUE THREAD

**Issue by PolarBean** [PrincetonUniversity/BrainPipe#4]
Hi, Is it possible to transform data which is registered to the Princeton Mouse Atlas into the Allen space?
Thanks in advance.

**Comment by tjp7rr** [comment]
Yes, you could use elastix (https://elastix.dev/) by itself or via brainpipe to compute the transformation. Attached are the two step affine, then bpsline transformation files between going from Allen to Princeton Mouse Atlas spaces. I would ensure you are applying the transform in the correct direction. Sometimes it's best to do the registration yourself in both directions as you'll have more control.

[ABA_to_PMA_Elastix_TransformParameters.0.txt](https://github.com/user-attachments/files/16839371/ABA_to_PMA_Elastix_TransformParameters.0.txt)
[ABA_to_PMA_Elastix_TransformParameters.1.txt](https://github.com/user-attachments/files/16839372/ABA_to_PMA_Elastix_TransformParameters.1.txt)

**Comment by PolarBean** [comment]
great, thanks! I am building a library to help others easily translate between brain atlases and plan on integrating the Princeton Atlas once I've replicated the transform. Thanks again for your help.

https://github.com/Neural-Systems-at-UIO/CCF_translator
